deploy/ocm-agent-operator-managedotifications-config/10-managednotifications-cr.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: ocmagent.managed.openshift.io/v1alpha1
+kind: ManagedNotifications
+metadata:
+  name: managednotifications
+  namespace: openshift-ocm-agent-operator
+spec:
+  notification:
+    - activeBody: |-
+      This is a notification when an alert is firing.
```

Comment:
Will need to change obviously but can be left until other things are addressed.

Reply:
```suggestion
      Your cluster requires you to take action as its ElasticSearch cluster logging deployment has been detected as reaching a high disk usage threshold. Red Hat SRE strongly recommends reducing application logging on your cluster to ensure logging continues to function. If logging disk consumption exceeds 95%, data will be at risk of becoming unavailable or lost and the stability of your ElasticSearch deployment may be impacted.
```

Reply:
Fixed